### ISSUE TITLE
300% CPU Usage?

### ISSUE THREAD

**Issue by prohtex** [lhc70000/iina#1722]
I'm seeing runaway CPU usage when using IINA to play a video under normal conditions. This has been causing my Macbook Pro to overheat and drain the battery very, very fast. Any ideas?

I am running v1.0.0-beta1.

<img width="1231" alt="screen shot 2018-05-21 at 5 45 47 pm" src="https://user-images.githubusercontent.com/12946938/40457756-07e58d76-5ead-11e8-97c9-05a77caaad07.png">

**Comment by saagarjha** [comment]
Please take a sample while this is occurring by clicking on the gear icon in the top left, then selecting "Sample process". Please provide the output this gives here so we can take a look into what's going on.

**Comment by lhc70000** [comment]
Moved to #1724.

tl;dr This may be fixed in beta 2.

**Comment by prohtex** [comment]
This has nothing to do with subtitles or txt files!

**Comment by prohtex** [comment]
I will post a sample soon

**Comment by lhc70000** [comment]
> I am also seeing the issue referenced in another ticket where CPU usage stays at 100% even when nothing is open.

Does this happen when you just launched IINA and before opening any video?

**Comment by prohtex** [comment]
No, only after

**Comment by lhc70000** [comment]
Seems that it's the problem addressed in #1724. Please wait for 1.0.0 beta2 and test again.

**Comment by prohtex** [comment]
Ok, I will test when it becomes available. The open video doesn't have any txt files, though.
